lib/jsonapi/plugs/format_required.ex

```diff
@@ -12,6 +12,57 @@ defmodule JSONAPI.FormatRequired do
 
   def call(%{method: method} = conn, _opts) when method in ~w[DELETE GET HEAD], do: conn
 
+  def call(
+        %{method: "POST", params: %{"data" => %{"type" => _, "relationships" => relationships}}} =
```

Comment:
I initially limited this to `POST` because the docs seem pretty clear about the spec/format for POST requests.

From <https://jsonapi.org/format/#crud-creating>:
> If a relationship is provided in the relationships member of the [resource object](https://jsonapi.org/format/#document-resource-objects), its value MUST be a relationship object with a data member. The value of this key represents the [linkage](https://jsonapi.org/format/#document-resource-object-linkage) the new resource is to have.

Whereas PATCH was a lot more indirect. It talks about "relationship object with a data member" which ultimately, in another section of the docs, translates to a resource linkage. So I'll have to update this to support both request types.

At first I was worried about being overly strict because I thought the PATCH left it open to pass stuff other than resource links. I didn't realize "relationship with a data member" was the same as the resource linkage stuff directly linked in the create section.

Future/updated version should address this.